src/Icicle/Sea/FromAvalanche/Program.hs

```diff
@@ -32,7 +32,7 @@ import           Icicle.Sea.FromAvalanche.Prim
 import           Icicle.Sea.FromAvalanche.State
 import           Icicle.Sea.FromAvalanche.Type
 
-import           P
+import           P hiding (head)
```

Comment:
I'm not sure. is it safe head? maybe it's just re-exported with a warning like error. hmm..